File: data/snippets/articles/guides/networking/rest/example_2.cpp
#include "splashkit.h"
#include <string.h>

using namespace std;

int main()
{
    http_response get_data;
    string response;

    // Get a list of a JSON web resource
    get_data = http_get("https://jsonplaceholder.typicode.com/posts/", 443);
    response = http_response_to_string(get_data);
    free_response(get_data);
    // To access each JSON key value pair the string should be split to an vector<json>
    // objects for simplicity sake we output just the string here.
    write_line(response);

    return 0;
}

File: data/snippets/articles/guides/networking/rest/example_2.py
import requests

# Get a list of JSON web resources
response = requests.get('https://jsonplaceholder.typicode.com/posts/')

# Output the response
if response.status_code == 200:
  for i in response.json():
    for key, value in i.items():
      print("{0} => {1}".format(key, value))
